### DISCUSSION TITLE
Hide all previews

### DISCUSSION THREAD

**Discussion by Doyley3731** [EveOPlus/eve-o-preview#5] [Ideas]
Hi there.

I've only been using this tool for a few days so my apologies if this idea.

If I have several clients open, the preview takes up quite a bit of real estate on my monitor.  I have them resized as small as possible but still big enough to be useful.  Sometimes I need to see or click on something that's behind.  I would love to have a hot key that hides the previews or toggles the "Always on top" option so I can quickly show/hide the previews allowing me to click things behind it without moving the clilent.

Is this possible?

Kind regards, and keep up the good work!

**Comment by EveOPlus** [comment]
This has been added in the latest releases, with a hotkey, button in UI, and right click menu on the thumbnail. Sorry for late reply. I didn't even know github had discussions
